### ISSUE TITLE
Move away from MemoryStore session store

### ISSUE THREAD

**Issue by ryanjdew** [marklogic/demo-cat#201]
MemoryStore isn't meant for production use. We should look into moving to another session store.

**Comment by grtjn** [comment]
@withjam has a good alternative if I am not mistaken.. :)

**Comment by withjam** [comment]
Yes, I made an implementation of a connect session store that uses MarkLogic to store sessions.  https://github.com/withjam/connect-marklogic

**Comment by ryanjdew** [comment]
That is great @withjam. Should we be fine using the same ML REST app as DemoCat or would you recommend creating a separate REST app and database for node.js sessions?

**Comment by withjam** [comment]
Session data is very small and it goes into it's own collection, so I don't think it would need a separate REST app unless you have a compelling reason to do so.  The only think to keep in mind is that the middleware itself doesn't take care of clearing expired sessions, so it may be something to monitor or clean up at some point.
